test/cpp/end2end/grpclb_end2end_test.cc

```diff
@@ -58,6 +58,8 @@
 // - Test handling of creation of faulty RR instance by having the LB return a
 //   serverlist with non-existent backends after having initially returned a
 //   valid one.
+// - test using secure credentials and make sure we don't send call
```

Comment:
This is done in https://github.com/grpc/grpc/pull/14507

Reply:
Thanks.  I'll merge this as-is now and let you remove this TODO as part of your PR.